libsquoosh/src/worker_pool.ts

```diff
@@ -1,13 +1,15 @@
 import { Worker, parentPort } from 'worker_threads';
+// @ts-ignore
```

Comment:
I think `TransformStream` should already have types. Did you encounter any errors while using it?

Reply:
![image](https://user-images.githubusercontent.com/229881/127586449-e24fea3f-8bd0-43dd-a950-4d228460f8ad.png)

Reply:
That is so strange, I do not receive that error. Did you check if you installed the dependencies and can you also check if the `build` fails with the same error?

Reply:
`npm run build` works fine, but VS Code is not happy

If you hit F12 (Go To Definition), where does it take you?

Reply:
It goes to here for me: `<squoosh-path>/libsquoosh/node_modules/web-streams-polyfill/dist/types/ts3.6/polyfill.d.ts`

Reply:
I tried several different ways to install and I never get a `./libsquoosh/node_modules` directory.

- `npm install`
- `npm ci`
- `yarn install`

Then I realized that workspaces aren't set up so I did `cd ./libsquoosh && npm ci` and that worked 😅